dbkit/dbkit_create.xml

```diff
@@ -1,37 +1,35 @@
-<tool id="dbkit_create" name="DBKit Create" version="0.1.1" python_template_version="3.5" license="MIT">
+<tool id="dbkit_create" name="DBKit Create" version="0.1.1" license="MIT" profile="20.01">
     <description>database</description>
-     <requirements>
-        <requirement type="package" version="3.2">python-wget</requirement>
-    </requirements>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
     <command detect_errors="exit_code"><![CDATA[
-        #if str($source.type) == "false":
-            mkdir -p inputs &&
-            #for item in $source.inputs
-                cp '${str(item)}' 'inputs/${item.element_identifier}' &&
-            #end for
-        #end if
-        python3 '$__tool_directory__/dbkit_create.py' -l '$input' -u '$source.url' -p 'inputs' -o '$index' -d '$database' -g '$log' -il '$advanced.idlength' -ic '$advanced.idcase' -ie '$advanced.idextension'
-    ]]></command>
+        mkdir -p inputs &&
+        #for item in $inputs
+            ln -s '${item}' 'inputs/${item.element_identifier}' &&
+        #end for
+        python3 '$__tool_directory__/dbkit_create.py'
+        -l '$input'
+        -p 'inputs'
+        -o '$index'
+        -d '$database'
+        -g '$log'
+        -il '$advanced.idlength'
+        -ic '$advanced.idcase'
+        -ie '$advanced.idextension'
+    ]]>    </command>
     <inputs>
-        <param format="tabular" name="input" type="data" label="List of Identifiers" help="Specify a list containing all entry identifiers to be downloaded."/>    
-        <conditional name="source">
```

Comment:
@guerler I completely removed the download part ... Galaxy should imho download it into a collection or whatever and this should should not be concerned about downloading ... ok?